### ISSUE TITLE
Pop!_Shop categories are broken when using default GNOME theme

### ISSUE THREAD

**Issue by tosziro** [pop-os/shop#215]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os

 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
Pop!_Shop

**Issue/Bug Description:**

Categories looks very bad when using default GNOME theme - Adwaita.

**Expected behavior:**

Clean looking categories

**Other Notes:**

Could you please guys fix this??

**Comment by simonsaysthis** [comment]
Might be trivial to some but its the reason I don't use Pop anymore. I just want the store to work with Adwaita.

**Comment by cassidyjames** [comment]
We could ship fallback color definitions in the styles, so it would work in any stylesheet—as long as the system stylesheet can still override those color definitions.

**Comment by simonsaysthis** [comment]
Thank you @cassidyjames that would be wonderful

**Comment by morgenman** [comment]
Excited to have this fixed

**Comment by simonsaysthis** [comment]
> We could ship fallback color definitions in the styles, so it would work in any stylesheet—as long as the system stylesheet can still override those color definitions.

Thanks again for fixing this so quickly

**Comment by morgenman** [comment]
This is fixed? I just updated my system and it was still broken.

**Comment by TechnologyMan101** [comment]
Well, it got fixed at least on my computer. See screenshot below.

![Screenshot from 2020-09-29 14-41-00](https://user-images.githubusercontent.com/65311278/94619461-cf6c5a00-0261-11eb-8879-2b814971f495.png)

**Comment by morgenman** [comment]
Did you build from source? I was waiting for it to get pushed to my pc automatically, but maybe you did it manually?

**Comment by morgenman** [comment]
Yea just tested, works for matcha, none of my other themes (even adwaita)

**Comment by TechnologyMan101** [comment]
Yeah, matcha before I updated it basically two days ago to fix a problem with the power menu (I installed it on Friday originally from GitHub) failed in Pop!_Shop. Now it works. Weird. I just tested with Adwaita and it fails. I guess we have to wait until the System76 teams pulls the update from Elementary.

**Comment by morgenman** [comment]
My theme pulls from /usr/share/themes too. I honestly think matcha just planned to support elementary.

**Comment by morgenman** [comment]
I fixed this on my machine by following jnsh's advice on this issue:
https://github.com/jnsh/arc-theme/issues/83

**Comment by TechnologyMan101** [comment]
I guess we'll have to wait for someone to create a pull request bringing in the changes. Also, icons really should have a fallback option too as the finance icon breaks when using another icon theme.

**Comment by morgenman** [comment]
Yea, there's a ton of icons that don't work for me. A generic fallback
theme would be good.

**Comment by TechnologyMan101** [comment]
Can you create a new issue for this?

I'm really busy and I can't even maintain my own GitHub stuff on time.
